### ISSUE TITLE
site: links to older versions of osquery packages

### ISSUE THREAD

**Issue by muffins** [facebook/osquery#3796]
We host all of the osquery packages at `https://pkg.osquery.io/[DISTRO TYPE]/osquery-VERSION.PACKAGE_EXT`. This works great for the latest versions, but we've had some folks ask for the last couple of releases to be made available as well, not just the latest. We need to extend our downloads website (https://osquery.io/downloads/) to contain links to these older packages, which are still hosted in our amazon s3 bucket.

An example of the latest package links is below:
```
https://pkg.osquery.io/linux/osquery-2.9.0_1.linux_x86_64.tar.gz
94684419037709386dcaa42e5b519e36370bb640d230f8d3852fb66fe1e23bb0
https://pkg.osquery.io/deb/osquery_2.9.0_1.linux.amd64.deb
898cd16b1df9bc7aeef91542611b0e209ff7e8b0aae273fafbf5ef9f77ed0037
https://pkg.osquery.io/darwin/osquery-2.9.0.pkg
83541f1241a8b92eee9cfa3dd32f384a2fd1c7ca639f138e3ec716e7ec5177a3
https://pkg.osquery.io/rpm/osquery-2.9.0-1.linux.x86_64.rpm
411ec4e7d84ed65e3915993824e424e54aeb358e60596059b268b27f3720b5e3
```

For this issue, extend the website to include the last three releases for Linux and Darwin for starters.

**Comment by micheal-o** [comment]
I am working on this.

**Comment by mjalkio** [comment]
@micheal-o I'm sorry, I didn't comment to say I'd be working on it. I just posted a PR :(

**Comment by micheal-o** [comment]
It's okay.

**Comment by muffins** [comment]
This was addressed in #3799
